build.sbt

```
@@ -13,12 +13,8 @@ resolvers ++= Seq(
   Resolver.sonatypeRepo("releases")
 )
 
-val defaultVersions = Map("chisel3" -> "3.3-SNAPSHOT")
-
-// Provide a managed dependency on chisel if -DchiselVersion="" issupplied on the command line.
-libraryDependencies ++= (Seq("chisel3").map {
-  dep: String => "edu.berkeley.cs" %% dep % sys.props.getOrElse(dep + "Version", defaultVersions(dep))
-})
+addCompilerPlugin("edu.berkeley.cs" % "chisel3-plugin" % "3.5.0-SNAPSHOT" cross CrossVersion.full)
+libraryDependencies += "edu.berkeley.cs" %% "chisel3" % "3.5.0-SNAPSHOT"
```

```suggestion
val chiselVersion = "3.5.0-RC2"
addCompilerPlugin("edu.berkeley.cs" % "chisel3-plugin" % chiselVersion cross CrossVersion.full)
libraryDependencies += "edu.berkeley.cs" %% "chisel3" % chiselVersion
```
I'd use 3.5.0-RC2, I also think we should move rocket-chip this direction

---

Sure.

---

I'd like to keep `"edu.berkeley.cs" % "chisel3-plugin" % "3.5.0-RC2" cross CrossVersion.full`, I think if adding scala-steward in the future, it cannot regex this pattern.